package/lib/src/utils/cupertino_icons.dart

```diff
@@ -0,0 +1,1349 @@
+import 'package:flutter/cupertino.dart';
+
+// Bash script to generate icons list
+//
+/*
+
+url='https://raw.githubusercontent.com/flutter/flutter/master/packages/flutter/lib/src/cupertino/icons.dart'
```

Comment:
Take icons from `stable` branch. We build Flet client against `stable` (3.16 at the moment):

```
https://raw.githubusercontent.com/flutter/flutter/stable/packages/flutter/lib/src/cupertino/icons.dart
```